rholang/reference_doc/bundles/README.md

```diff
@@ -1,7 +1,22 @@
 ## Bundles
+Bundles can be used to control the read/write privileges on a channel
 
-- Description of bundles
-  - When can be used
-- Good examples
+### Write only
+- `bundle+{ chan }`
+- `chan` is write only
 
-I think this will be somewhat nuanced, so careful exposition here.
+### Read only
+- `bundle-{ chan }`
+- `chan` is read only
+
+### No read, no write
+- `bundle0{ chan }`
+- `chan` can't be read or written on
+- Useful in some cases
+  - EG here
+
+### bundle: bundle read write
+- `bundle-{ chan }`
```

Comment:
@differentialderek I think this is the line that @goral09 Was commenting on. The `-` here is accidental, right?

Reply:
Ah, yes, thank you.